src/firecrest/filesystem/transfer/router.py

```diff
@@ -470,6 +470,7 @@ async def post_cp(
         ),
         "source_path": request.path,
         "target_path": request.target_path,
+        "dereference": "L" if request.dereference else "d",
```

Comment:
The "L" and "d" parameter are specific to the copy command logic.
You should move them into the slurm_job_copy.sh script